Dockerfile

```diff
@@ -1,5 +1,49 @@
+FROM ubuntu:16.04 as builder
+
+RUN apt-get update \
+    && apt-get install -y curl git wget locales python-pip make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev apt-transport-https libpq-dev
+
+RUN locale-gen en_US.UTF-8
+
+ENV LANG=en_US.UTF-8
+
+RUN git clone git://github.com/yyuu/pyenv.git .pyenv
+RUN git clone https://github.com/yyuu/pyenv-virtualenv.git ~/.pyenv/plugins/pyenv-virtualenv
+
+ENV HOME  /
+ENV PYENV_ROOT $HOME/.pyenv
+ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+
+RUN pyenv install 3.4.2
```

Comment:
We should lean on the `.python_version` file

Reply:
I've spent quite a while playing with this. `pyenv` seems to cause a lot of problems in docker, so I'm going to stick with the python base container for now.